WcaOnRails/app/models/registration.rb

```diff
@@ -33,10 +33,6 @@ def accepted?
     !pending?
   end
 
-  def events
-    registration_events.reject(&:marked_for_destruction?).map(&:event_object).sort_by(&:rank)
-  end
```

Comment:
I thought we got rid of all `event_object` methods, but it looks like one left (we don't need it as it does the same as `event` since event bacame a real model). Feel free to remove this from the Event model.